src/Sql/Sql/ChangeLog.md

```diff
@@ -27,6 +27,7 @@
 ## Version 2.1.1
 * Update references in .psd1 to use relative path
 * Upgraded storage creation in Vulnerability Assessment auto enablement to StorageV2
+* Adding support for changing existing managed instance hardware generation
```

Comment:
Please rebase the master and add this new changelog line understand upcoming release